storage-access.bs

```diff
@@ -293,6 +308,49 @@ ISSUE: Write this section.
 
 ISSUE: Write this section.
 
+<h2 id="automation">Automation</h2>
+
+For the purposes of user-agent automation and application testing, this document defines the following [=extension command=] for the [[WebDriver]] specification.
+
+<h3 id="set-storage-access-command">Set Storage Access</h3>
+
+<table>
+    <tbody>
+    <tr>
+        <th>HTTP Method</th>
+        <th>URI Template</th>
+    </tr>
+    <tr>
+        <td>POST</td>
+        <td>/session/{session id}/storageaccess</td>
+    </tr>
+    </tbody>
+</table>
+
+The <dfn export>Set Storage Access</dfn> [=extension command=] modifies the storage access policy for the [=current browsing context=].
+
+The [=remote end steps=] are:
+
+
+1. Let |blocked| be the result of [=getting a property=] from |parameters| named `blocked`.
+1. If |blocked| is not a [=boolean=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. Let |origin| be the result of [=getting a property=] from |parameters| named `origin`.
+1. If |origin| is not the [=serialization of an origin=] return a [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
```

Comment:
This doesn't work. I guess what you want to be doing is parse this field as a URL and then use that URL's origin. Similar to `postMessage()`. That would also help the following steps which assume origin is an object (and not a string).

Reply:
Thanks. I've adapted some of the language from the spec for `postMessage`. I've also added similar handling for asterisk (*) to provide a mechanism to specify all origins per @Brandr0id's request above.